### PR TITLE
Match multiline translation expressions in modules

### PR DESCRIPTION
**Pull request by unlocomqx**
When a module developers write a multiline translation expression, it's not displayed in the backoffice translation page.
This modification matches white spaces (if any) to allow for more flexible detection.
Example:
```
$this->l(
  'Are you sure you want to delete this customization?',
  $source
);
```

Or

```
$this->l(
  'Are you sure you want to delete this customization?'
);
```

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | match multiline translation expressions
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/12903)

### CODE REVIEW

**Comment by unlocomqx** [comment]
This should fix https://github.com/PrestaShop/PrestaShop/issues/11573

File: controllers/admin/AdminTranslationsController.php [hunk 1]
```diff
@@ -1275,7 +1275,7 @@ protected function userParseFile($content, $type_translation, $type_file = false
             case 'modules':
                 // Parsing modules file
                 if ($type_file == 'php') {
-                    $regex = '/->l\((\')' . _PS_TRANS_PATTERN_ . '\'(, ?\'(.+)\')?(, ?(.+))?\)/U';
+                    $regex = '/->l\(\s*(\')' . _PS_TRANS_PATTERN_ . '\'(\s*,\s*?\'(.+)\')?(\s*,\s*?(.+))?\s*\)/U';
```
Comment (PierreRambaud) [inline controllers/admin/AdminTranslationsController.php]: You can also add `ms`  (http://php.net/manual/en/reference.pcre.pattern.modifiers.php)

Comment (unlocomqx) [reply controllers/admin/AdminTranslationsController.php]: The modifiers are now set to `Ums` in a6e8796ba1356ce6ade0b9a745b81be22032bcc0

**Review by PierreRambaud — DISMISSED** [review]

**Comment by PierreRambaud** [comment]
Hey sorry for the delay.
Are you able to provide us a module for our QA team to help us to reproduce?

**Comment by unlocomqx** [comment]
No worries, here's a sample module to reproduce
[trans.zip](https://github.com/PrestaShop/PrestaShop/files/3386634/trans.zip)

Before fix


After fix

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @unlocomqx

**Pull request merged**